tests/spec/path-resolution.spec.js

```diff
@@ -241,7 +242,7 @@ describe('path resolution', function() {
   });
 
   it('should properly remove trailing slashes with Path.removeTrailing()', function() {
-    var Path = Filer.Path;
+    let Path = Filer.Path;
```

Comment:
```suggestion
    const Path = Filer.Path;
```
Great job with the code, I just think that since path wont be changing later on, you should put it const instead of let.